external/vulkancts/modules/vulkan/texture/vktTextureShadowTests.cpp

```diff
@@ -1904,6 +1922,24 @@ void populateTextureShadowTests (tcu::TestCaseGroup* textureShadowTests)
 #endif // CTS_USES_VULKANSC
 }
 
+namespace util
```

Comment:
There was feedback from the internal review that it appears these two functions are unused. If that's the case, can you remove them. Thanks.

Reply:
This probably broke because the branch sat around for half a year, should be fixed now.